editor/editor_node.cpp

```diff
@@ -521,10 +522,21 @@ void EditorNode::_update_theme(bool p_skip_creation) {
 		help_menu->set_item_icon(help_menu->get_item_index(HELP_ABOUT), theme->get_icon(SNAME("Godot"), EditorStringName(EditorIcons)));
 		help_menu->set_item_icon(help_menu->get_item_index(HELP_SUPPORT_GODOT_DEVELOPMENT), theme->get_icon(SNAME("Heart"), EditorStringName(EditorIcons)));
 
+		for (int i = 0; i < main_editor_buttons.size(); i++) {
```

Comment:
There is a MainScreenButton variation added recently, which is used by main editor buttons. You should modify it instead of adding overrides.
(note that it's also used in Project Manager)